src/views/user/Registration.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div>
+    <form>
+      <b-row class="mb-3">
+        <h4>{{ $t('navbar.registration') }}</h4>
+      </b-row>
+
+      <b-row class="mb-3">
+        <b-col cols="2">{{ $t('navbar.forename') }}</b-col>
+        <b-col cols="2"><b-form-input v-model="forename"></b-form-input></b-col>
+      </b-row>
+
+      <b-row class="mb-3">
+          <b-col cols="2">{{ $t('navbar.surname') }}</b-col>
+          <b-col cols="2"><b-form-input v-model="surname"></b-form-input></b-col>
+      </b-row>
+      
+      <b-row class="mb-3">
+          <b-col cols="2">{{ $t('navbar.email') }}</b-col>
+          <b-col cols="2"><b-form-input v-model="email" type="email"></b-form-input></b-col>
+      </b-row>
+      
+      <b-row class="mb-3">
+          <b-col cols="2">{{ $t('navbar.password') }}</b-col>
+          <b-col cols="2"><b-form-input v-model="password" type="password"></b-form-input></b-col>
+      </b-row>
+      
+      <b-row class="mb-3">
+          <b-col cols="2">{{ $t('navbar.repassword') }}</b-col>
+          <b-col cols="2"><b-form-input v-model="repassword" type="password"></b-form-input></b-col>
```

Comment:
Let's style this just a bit when repassword is not null and repassword != password.  Maybe a little red dot beside the field, an maybe grey text instead of black?  That can help a lot when people don't know why they cannot click on "register".

Reply:
We will write a small message: "Passwords must be identical"